pipelines/azure-pipelines.docker.yml

```diff
@@ -21,4 +21,4 @@ steps:
     scriptType: bash
     scriptLocation: inlineScript
     inlineScript: |
-      az acr build -t $(TF_VAR_JMETER_DOCKER_IMAGE) -r $(TF_VAR_JMETER_IMAGE_REGISTRY_NAME) -f $(Build.SourcesDirectory)/docker/Dockerfile $(Build.SourcesDirectory)/docker
\ No newline at end of file
+      az acr build -t $(TF_VAR_JMETER_DOCKER_IMAGE) -r $(TF_VAR_JMETER_IMAGE_REGISTRY_NAME) -f $(Build.SourcesDirectory)/docker/Dockerfile $(Build.SourcesDirectory)/docker
```

Comment:
@allantargino I saw on the #45 that `TF_VAR_JMETER_ACR_NAME` variable will be renamed to `TF_VAR_JMETER_ACR_NAME`.  I'm wondering if the PR #45 will break this pipeline when it's merged.

Reply:
Nevermind, I just saw the variable name will be changed here https://github.com/Azure-Samples/jmeter-aci-terraform/pull/45/files#diff-451a4bcfc7487d45b84628860214f3f1R24. The pipeline won't break 👍